conbench/api/results.py

```diff
@@ -266,7 +266,7 @@ def post(self) -> f.Response:
     methods=["GET", "POST"],
 )
 rule(
-    "/benchmark=results/<benchmark_result_id>/",
+    "/benchmark-results/<benchmark_result_id>/",
```

Comment:
Thank you! Sorry for missing that; I know we talked about this elsewhere and I should have carried this into a TODO list. Perfect!